R/rope.R

```diff
@@ -13,17 +13,19 @@
 #'   different from a single point being infinite). Therefore, the idea
 #'   underlining ROPE is to let the user define an area around the null value
 #'   enclosing values that are equivalent to the null value for practical
-#'   purposes (\cite{Kruschke 2010, 2011, 2014}). Kruschke (2018) suggests that
-#'   such null value could be set, by default, to the -0.1 to 0.1 range of a
-#'   standardized parameter (negligible effect size according to Cohen, 1988).
-#'   This could be generalized: For instance, for linear models, the ROPE
-#'   could be set as \code{0 +/- .1 * sd(y)}. Kruschke (2010, 2011, 2014) suggests
-#'   using the proportion of the 95\% (or 90\%, considered more stable)
-#'   \link[=hdi]{HDI} that falls within the ROPE as an index for
-#'   "null-hypothesis" testing (as understood under the Bayesian framework,
-#'   see \link[=equivalence_test]{equivalence_test}). Besides the ROPE-based
-#'   decision criteria, the proportion of the 95\% CI that falls in the ROPE
-#'   can be used as a continuous index.
+#'   purposes (\cite{Kruschke 2010, 2011, 2014}).
```

Comment:
@DominiqueMakowski I'm not sure where you are referring to, could you please add the references in APA-style here?